### PR TITLE
Added in static files for HeronUI so that Offline Heron Deployment works

### PR DESCRIPTION
**Pull request by mjschmidt**

### CODE REVIEW

**Comment by cckellogg** [comment]
Can we add versions to the  js file names? 

Also, there a few more dependencies:
//cdnjs.cloudflare.com/ajax/libs/d3/3.4.11/d3.min.js
//cdnjs.cloudflare.com/ajax/libs/d3-tip/0.6.3/d3-tip.min.js

See files:
config.html
exception.html
topologies.html
topology.html

**Comment by mjschmidt** [comment]
I added the js files from the links they were being called from, added the files names and replaced the cloudera links in:
application.html
config.html
exception.html
topologies.html
topology.html

**Comment by mjschmidt** [comment]
YAY it worked (:

**Comment by kramasamy** [comment]
Awesome!

File: heron/tools/ui/resources/templates/application.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
 
     <link href="{{ static_url('css/visstyle.css') }}" rel="stylesheet">
 
-    <script src="//cdnjs.cloudflare.com/ajax/libs/react/0.10.0/react.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/react/0.10.0/JSXTransformer.js"></script>
+    <script src="{{ static_url('js/react.0.10.0.js') }}"></script>
+    <script src="{{ static_url('js/JSXTransformer-0.10.0.jss') }}"></script>
```
Comment (cckellogg) [inline heron/tools/ui/resources/templates/application.html]: spelling mistake? should just be .js right?

**Pull request merged**